docs/reference/analysis/tokenizers/keyword-tokenizer.asciidoc

```diff
@@ -44,6 +44,47 @@ The above sentence would produce the following term:
 [ New York ]
 ---------------------------
 
+[float]
+=== Exmaple output(Lower-casing email address)
+
+[source,console]
+---------------------------
+POST _analyze
+{
+  "tokenizer": "keyword",
+  "filter": ["lowercase"],
+  "text": "john.SMITH@global-international.COM"
```

Comment:
Changing the spacing to be more consistent with our other examples.

Let's also use `example.com` as the placeholder domain.
```suggestion
  "filter": [ "lowercase" ],
  "text": "john.SMITH@example.COM"
```